docs/README.md

```diff
@@ -0,0 +1,181 @@
+# UserManagementService
```

Comment:
This README seems to be eligible to be present at root level.

Reply:
done